src/components/dashboard/containers/Tables/MUIDataTableEnhanced.js

```diff
@@ -1,78 +1,78 @@
-import React from 'react'
-import MUIDataTable from 'mui-datatables'
-import { t } from 'selectors'
-
-const generalTableOptions = {
-	rowsPerPage: 5,
-	rowsPerPageOptions: [5, 10, 25, 50, 100, 500],
-	setTableProps: () => {
-		return {
-			padding: 'default',
-			size: 'small',
-			tableLayout: 'auto',
-			width: '100%',
-			align: 'center',
-			whiteSpace: 'wrap',
-		}
-	},
-	textLabels: {
-		body: {
-			noMatch: t('TABLE_NO_MATCH'),
-			toolTip: t('TABLE_TOOLTIP'),
-			columnHeaderTooltip: column =>
-				`${t('TABLE_HEADER_TOOLTIP', {
-					args: [column.label || column],
-				})}`,
-		},
-		pagination: {
-			next: t('TABLE_NEXT'),
-			previous: t('TABLE_PREV'),
-			rowsPerPage: t('TABLE_ROWS_PER_PAGE'),
-			displayRows: t('TABLE_DISPLAY_ROWS'),
-		},
-		toolbar: {
-			search: t('TABLE_SEARCH'),
-			downloadCsv: t('TABLE_DL_CSV'),
-			print: t('TABLE_PRINT'),
-			viewColumns: t('TABLE_VIEW_COLS'),
-			filterTable: t('TABLE_FILTER'),
-		},
-		filter: {
-			all: t('TABLE_FILTER_ALL'),
-			title: t('TABLE_FILTER_TITLE'),
-			reset: t('TABLE_FILTER_RESET'),
-		},
-		viewColumns: {
-			title: t('TABLE_SHOW_COLS'),
-			titleAria: t('TABLE_SHOW_HIDE_COLS'),
-		},
-		selectedRows: {
-			text: t('TABLE_SELECTED_ROWS'),
-			delete: t('TABLE_DELETE'),
-			deleteAria: t('TABLE_DELETE_SELECTED'),
-		},
-	},
-}
-
-export default function MUIDataTableEnchanced(props) {
-	const { title, data, columns, options } = props
-
-	return (
-		<MUIDataTable
-			title={title}
-			data={data}
-			columns={columns}
-			options={{
-				...generalTableOptions,
-				...options,
-				search: !props.noSearch,
-				download: !props.noDownload,
-				print: !props.noPrint,
-				selectableRows: props.rowSelectable ? 'multiple' : 'none',
-				customToolbarSelect: (selectedRows, displayData, setSelectedRows) => {
-					//This disables toolbar when selected elements on all tables
-				},
-			}}
-		/>
-	)
-}
+import React from 'react'
+import MUIDataTable from 'mui-datatables'
+import { t } from 'selectors'
+
+const generalTableOptions = {
+	rowsPerPage: 5,
+	rowsPerPageOptions: [5, 10, 25, 50, 100, 500],
+	setTableProps: () => {
+		return {
+			padding: 'default',
+			size: 'small',
+			tableLayout: 'auto',
```

Comment:
I think this needs to be removed as well as the wrap. Had some errors in console and I think they are for this.